_data/build-research.py

```diff
@@ -0,0 +1,65 @@
+# script to build bibliography of papers on "research" page
+
+# before running:
+# install python          https://www.python.org/downloads/
+# install pip             https://pip.pypa.io/en/stable/installing/
+# install manubot         https://github.com/manubot/manubot#installation
+
+import os
+import json
+import yaml
+import subprocess
+import calendar
```

Comment:
```suggestion
import calendar
import datetime
```
if you use my suggestion below